### DISCUSSION TITLE
tun.routes based on the underlying network?

### DISCUSSION THREAD

**Discussion by migs35323** [slackhq/nebula#1236] [General]
i have a few set of nodes that share a connection in the same network switch
more specifically, a few nodes have 2 network interfaces, one for talk with each other and other for general internet to talk with external nodes
and could benefit from a huge MTU, however by default i did choose the 1300 mtu because most nodes are external in the internet.

when i tried to use the direct conneciton nic cidr as the tun.mtu value it gave me an error about the certificate, so i guess this setting is per nebula network..

So, i would still keep like to use nebula even tough they are in direct access (several reasons like encryption, portability, and an unique ip to rule them all)

**Comment by johnmaguire** [comment]
@migs35323 `tun.routes` allows you to specify an MTU to be used with specific VPN IP ranges. You are asking to be able to specify an MTU for specific underlay IP ranges, is that correct?
